jax_unirep/evotuning.py

```diff
@@ -379,22 +493,20 @@ def evotune(
         This controls the learning rate of the model.
         See source code for default configuration,
         at the definition of ``learning_rate_kwargs``.
+    :param n_splits: The number of folds of cross-validation to do.
+    :param steps_per_print: the number of steps between each print,
+        will print out current evotuned_params.
+
     :returns:
         - study - The optuna study object, containing information
         about all evotuning trials.
         - evotuned_params - A dictionary of optimized weights
     """
-    if params is None:
-        # _, params = init_fun(random.PRNGKey(0), input_shape=(-1, 10))
-        params = (load_params_1900(), (), load_dense_1900())
-
-        # params = dict()
-        # params["dense"] = load_dense_1900()
-        # params["mLSTM1900"] = load_params_1900()
+    # if params is None:
+    #     params = (load_params_1900(), (), load_dense_1900())
```

Comment:
Sorry about this, I should have just gone ahead and deleted these lines. This is a duplicated piece of code from the `fit` function that I wouldn't have caught without a test. (Tests are important!)

```suggestion
```